doc/source/example-a3c.rst

```diff
@@ -11,6 +11,10 @@ View the `code for this example`_.
 .. _`Universe Starter Agent`: https://github.com/openai/universe-starter-agent
 .. _`code for this example`: https://github.com/ray-project/ray/tree/master/python/ray/rllib/a3c
 
+.. note::
+
+    For a more comprehensive framework of reinforcement learning in Ray, see `Ray RLlib <http://ray.readthedocs.io/en/latest/rllib.html>`__.
```

Comment:
(Wording suggestions welcome)

Reply:
How about, "For an overview of Ray's reinforcement learning library, see ..."

Reply:
I like it. Updated.